include/envoy/http/access_log.h

```diff
@@ -10,29 +10,31 @@
 namespace Http {
 namespace AccessLog {
 
-enum class FailureReason {
+enum ResponseFlag {
   // No failure.
-  None,
+  None = 0x0,
```

Comment:
Where do you actually need to check if None is set. That location is probably now a bug in the face of something like DI.